source/partials/release_notes/_release-17-10-0.md.erb

```diff
@@ -0,0 +1,42 @@
+**Important:** After the 17.9 release, we made an announcement asking users to hold off upgrading their GoCD instance. We found an issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>) affecting a small set of users. This release provides a fix for the same.
+
+<h4>Critical Bug Fixes</h4>
+* <%= link_to_commit '05598d88fd4dabdde1184faa4fbffc5f9406d0dc', 'Unable to start the server after 17.9.0 upgrade', 3834%>
+
+<h4>Other Changes</h4>
+* <%= link_to_commit '176ebc26438784f50bbdc013ebc704a3a6cd6c8d', 'Fix for no plugins visible after 17.9.0 upgrade', 3829%>
+
+<h4>Docker Images</h4>
+* Allow users to specify `uid` and `gid` as build args while building a customize docker image from GoCD image (both server and agent).
+* Allow users to specify a volume mount instruction in docker run command.
+
+<h4>Contributors</h4>
```

Comment:
Add external contributors who have contributed to docker-gocd-server and docker-gocd-agent.